Source/EventAnalysis/KTIterativeTrackClustering.hh

```diff
@@ -0,0 +1,122 @@
+/**
+ @file KTIterativeTrackClustering.hh
+ @brief Contains KTIterativeTrackClustering
+ @details Groups collinear tracks into one
+ @author: C. Claessens
+ @date: August 7, 2017
+ */
+
+#ifndef KTITERATIVETRACKCLUSTERING_HH_
+#define KTITERATIVETRACKCLUSTERING_HH_
+
+#include "KTPrimaryProcessor.hh"
+
+#include "KTSlot.hh"
+#include "KTData.hh"
+#include "KTMemberVariable.hh"
+#include "KTProcessedTrackData.hh"
+
+#include <algorithm>
```

Comment:
I don't see that you've used all of these include files here (aside from std::vector).  If not, move them to the source file or remove them.